huxley/settings/registration.py

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) 2011-2014 Berkeley Model United Nations. All rights reserved.
+# Use of this source code is governed by a BSD License (see LICENSE).
+
+REGISTRATION_OPEN = False
```

Comment:
You'll also have to import this setting into the __init__.py file of this directory, otherwise it won't be available in `django.conf.settings`.